servers/cu/src/domain/index.js

```diff
@@ -185,7 +186,9 @@ export const createApis = async (ctx) => {
             value.evaluation,
             err
           )
-        })
+        }),
+    setTimeout: (...args) => lt.setTimeout(...args),
+    clearTimeout: lt.clearTimeout
```

Comment:
Same deal as above -- eta conversion may be needed, in case the dependency references `this`